src/main/java/parser/Parser.java

```diff
@@ -13,20 +14,9 @@
 import command.RecentlyAddedCommand;
 import command.SearchFrequencyCommand;
 import command.EditCommand;
+import command.AddSynonymCommand;
 
-import exception.CommandInvalidException;
-import exception.EmptyWordException;
-import exception.WordUpException;
-import exception.WrongAddFormatException;
-import exception.WrongDeleteFormatException;
-import exception.WrongSearchFormatException;
-import exception.WrongListFormatDescription;
-import exception.WrongHistoryFormatException;
-import exception.ZeroHistoryRequestException;
-import exception.WrongSearchFrequencyFormatException;
-import exception.WrongEditFormatException;
-import exception.WrongAddTagFormatException;
-import exception.WrongQuizFormatException;
+import exception.*;
```

Comment:
oh ya btw dun do the * import if not later gotta retype cuz it'll fail the checkstyle